docs/src/modules/components/DemoSandboxed.js

```diff
@@ -117,6 +117,9 @@ DemoFrame.propTypes = {
   name: PropTypes.string.isRequired,
 };
 
+// Use the default MUI theme for the demos
+const theme = createTheme();
```

Comment:
need to pass `mode` from upper theme.

Reply:
Good catch! Fixed with https://github.com/mui-org/material-ui/pull/27789/commits/1f84e6dd70c7547a175d225e8cd7c8b250e859bc